app/viewmodels/journeys/adjustments/NetBusinessProfitOrLossSummary.scala

```diff
@@ -0,0 +1,82 @@
+/*
+ * Copyright 2024 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package viewmodels.journeys.adjustments
+
+import models.journeys.adjustments.ProfitOrLoss
+import play.api.i18n.Messages
+import uk.gov.hmrc.govukfrontend.views.Aliases.Table
+import uk.gov.hmrc.govukfrontend.views.viewmodels.table.TableRow
+import viewmodels.checkAnswers.{buildTable, buildTableAmountRow}
+
+object NetBusinessProfitOrLossSummary {
+
+  private val placeholderAmount: BigDecimal = 200
+
+  def buildNetProfitTable(profitOrLoss: ProfitOrLoss)(implicit messages: Messages): Table = {
+
+    val turnover                   = placeholderAmount
+    val incomeNotCountedAsTurnover = placeholderAmount
+    val totalExpenses              = placeholderAmount
+    val netProfitOrLoss            = turnover + incomeNotCountedAsTurnover + totalExpenses
+
+    val netProfitRows: Seq[Seq[TableRow]] = Seq(
+      buildTableAmountRow("profitOfLoss.turnover", turnover),
+      buildTableAmountRow("incomeNotCountedAsTurnover.title", incomeNotCountedAsTurnover),
+      buildTableAmountRow("profitOfLoss.totalExpenses", totalExpenses),
+      buildTableAmountRow(s"profitOfLoss.netProfitOrLoss.$profitOrLoss", netProfitOrLoss)
+    )
+
+    buildTable(
```

Comment:
SRP: Single Responsible Principle - "every class, module, or function in a program should have one responsibility/purpose in a program."

here we're mixing calculations with the UI Table.

I suggest extracting  calculation to a separate function returning BigDecimal, which will be easier to refactor and test later.

Reply:
Have added a comment for whoever needs to do this in their ticket